packages/interface/src/contexts/Maci.tsx

```diff
@@ -410,15 +408,14 @@ export const MaciProvider: React.FC<MaciProviderProps> = ({ children }: MaciProv
 
   /// check the poll data and tally data
   useEffect(() => {
-    setIsLoading(true);
-
     // if we have the subgraph url then it means we can get the poll data through there
     if (config.maciSubgraphUrl) {
```

Comment:
check if the user provided the maci subgraph url here, do we need a callback reminding user to provide the url if they don't? (instead of fetching the contract data as a backup method, we just remind them to provide the url)